supply-chain/audits.toml

```diff
@@ -0,0 +1,558 @@
+
```

Comment:
We try to keep the top level directory clean in this repository. Would be great to move these files to `misc/cargo-vet`. (I think `cargo-vet` is more descriptive as a directory name than `supply-chain`.) You can point `cargo-vet` at the location by adding a block like this to the root Cargo.toml:

```
[package.metadata.vet]
store = { path = "misc/cargo-vet" }
```